pkg/kv/dist_sender.go

```diff
@@ -471,6 +450,7 @@ func (ds *DistSender) sendSingleRange(
 
 	br, err := ds.sendRPC(ctx, desc.RangeID, replicas, ba)
 	if err != nil {
+		log.ErrEvent(ctx, err.Error())
```

Comment:
Was this for debugging, or is this error just not otherwise captured in the trace further up the call stack?